examples/funcalign/rsrm_synthetic_reconstruction.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## This example replicates the synthetic experiment in the paper\n",
+    "## \"Capturing Shared and Individual Information in fMRI Data\", J. Turek, C. Ellis, L. Skalaban, N. Turk-Browne, T. Willke"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Import some libraries that we will need"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import scipy.io\n",
+    "from scipy.stats import stats\n",
+    "import numpy as np\n",
+    "import brainiak.funcalign.srm as srm\n",
+    "import brainiak.funcalign.rsrm as rsrm\n",
+    "from mpl_toolkits.mplot3d import Axes3D\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Experiment setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "voxels = 100\n",
+    "samples = 200\n",
+    "subjects = 10\n",
+    "features = 3\n",
+    "snr = 20 # in dB\n",
+    "amplitude = 8.0 # approximate amplitude used for the values in Si\n",
+    "p = 0.05 # probability of being non-zero in Si"
```

Comment:
How does this relate to the gamma value used later? Why are they not the same?

Reply:
p is the probability of an element to be non-zero in the *original* Si (drawn from the model).
gamma is selected to approximately have the same number of non-zeros in the *estimated* Si.
However, there is no reason for them to have the same values. Ones is the probability and the other is a balancing value between regularization terms.

Reply:
I might still be misunderstanding but shouldn't the non-zero elements of the original Si be thought of as the individual component that RSRM is aiming to extract? Why would there be a different number in the input vs the output?

Reply:
The math is not translated 1:1. Remember that this is probability, and when you draw a signal there are not exactly 5% of non-zeros. There could be anywhere near the 0.05. Added to that this will be differently with all subjects. So every subject will have approximately that, but not exactly. 
Therefore, even if you can obtain a 1:1 translation from p (range 0 to 1) to gamma (range 0 to +inf), still the signal itself will not necessarily behave in that way. 

Also, this is a particular case of drawing Si's, and you can change it by other distributions as well. In such case, you will have parameters to generate the signals and still gamma to recover it. Dividing between gamma and p, it is a good idea.

Reply:
Okay fair enough I understand